build.gradle

```diff
@@ -5,7 +5,7 @@ plugins {
 }
 
 group 'uk.gov.hmcts.reform'
-version '1.1.3'
+version '1.1.4'
```

Comment:
I think this should be at least a minor bump rather than a patch

Reply:
100%. It's not a major bump because we don't break anything, its not a patch release because we're not fixing bugs. We're adding new features so it should be a minor bump (ie, 1.2.0)

Reply:
agreed and resolved